apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -1,36 +1,109 @@
 import { html } from 'lit';
-
+import { ifDefined } from 'lit/directives/if-defined.js';
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-checkbox';
 import { CheckboxProps } from '@justeattakeaway/pie-checkbox';
 /* eslint-enable import/no-duplicates */
 
+import { action } from '@storybook/addon-actions';
 import { type StoryMeta } from '../types';
 import { createStory } from '../utilities';
 
 type CheckboxStoryMeta = StoryMeta<CheckboxProps>;
 
-const defaultArgs: CheckboxProps = {};
+const defaultArgs: CheckboxProps = {
+    value: '',
+    name: '',
+    label: 'Label',
+    disabled: false,
+    checked: false,
+    indeterminate: false,
+};
 
 const checkboxStoryMeta: CheckboxStoryMeta = {
     title: 'Checkbox',
     component: 'pie-checkbox',
-    argTypes: {},
+    argTypes: {
+        value: {
+            description: 'The value of the checkbox (used as a key/value pair in HTML forms with `name`).',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        name: {
+            description: 'The name of the checkbox (used as a key/value pair with `value`). This is required in order to work properly with forms.',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        label: {
+            description: 'The visible label for the checkbox',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        checked: {
+            description: 'Indicates whether or not the checkbox is checked by default (when the page loads).',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+        disabled: {
+            description: 'If true, disables the checkbox.',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+        indeterminate: {
```

Comment:
This is super interesting – I had seen this state used but didn't know the full use case of it until now ([and read this article too](https://css-tricks.com/indeterminate-checkboxes/)).

Reply:
Was asking myself what `indeterminate` was for a few seconds ago! 😄